web-api/elasticsearch/efcms-message-mappings.js

```diff
@@ -1,5 +1,11 @@
 module.exports = {
   properties: {
+    case_relations: {
+      relations: {
+        case: 'message',
+      },
+      type: 'join',
+    },
     'caseStatus.S': {
```

Comment:
I thought we would need to add leadDocketNumber to this message-mappings.js file.  I think we did something similar for docket-entry-mapping.js where we had to put `caseCaption` into that mapping to allow us to query for it in the future.  Just a thought... if everything seems to be working and tests are good, maybe it's not an issue.